test/core/bad_ssl/server_common.cc

```diff
@@ -28,19 +29,23 @@
  * configured to cause some failure case in the SSL connection path.
  */
 
+// In some distros, gflags is in the namespace google, and in some others,
+// in gflags. This hack is enabling us to find both.
+namespace google {}
```

Comment:
Rather than duplicate this in all of the files, use a common headers that has this comment and namespace hack

Reply:
Would suggest adding to test/core/util/test_config.h actually. Just like is done in test/cpp/util/test_config.h